rosetta/src/CMakeLists.txt

```diff
@@ -2,16 +2,16 @@ set(_common_sources
     "rosetta-common.cpp"
     "rosetta-stat.cpp"
     "cdflib.cpp"
-    "stdtr.c"
-    "ndtr.c"
-    "ndtri.c"
-    "incbet.c"
-    "incbi.c"
-    "mtherr.c"
-    "const.c"
-    "gamma.c"
-    "polevl.c"
-    "igam.c")
+    "stdtr.cxx"
```

Comment:
Could you use the extension `.cpp` like the other files in the directory?

Reply:
Fixed